src/notation/view/widgets/editstyle.cpp

```diff
@@ -1088,6 +1148,51 @@ void EditStyle::setPage(int idx)
     }
 }
 
+//---------------------------------------------------------
+//   pageListRowChanged
+//---------------------------------------------------------
+
+void EditStyle::pageListRowChanged(int row)
+{
+    pageStack->setCurrentIndex(pageListMap[row]);
+}
+
+//---------------------------------------------------------
+//   pageListMoved
+//---------------------------------------------------------
+
+void EditStyle::pageListMoved(QModelIndex,int Start,int,QModelIndex,int End)
+{
+    if (End>Start) {
+        int startPageIndex = pageListMap[Start];
+        for (int i=Start;i<(End-1);i++) {
```

Comment:
spaces agter the ; and bore and after the ", the y< and the -

Reply:
ok .. that's a lot of similar things before and after the line ...